### ISSUE TITLE
[Feature][iOS] Support for saving / creating live photo

### ISSUE THREAD

**Issue by ua741** [fluttercandies/flutter_photo_manager#805]
**Is your feature request related to a problem? Please describe.**
This request is related to #386. 
The library allows you to save the video and image. There should be a third option (iOS/MacOS(?) only) to save a live photo.

**Describe the solution you'd like**
Support for saving live photo
```dart
Future<AssetEntity?> result = PhotoManager.editor.saveLivePhoto({@required File image, @required File video ...});
```
**Describe alternatives you've considered**
1) Save video and image assets separately.
This option make sense for Android/Windows device, but iOS users will prefer to save them as a single asset.

2) Use [live_photos](https://pub.dev/packages/live_photos)
This generates the photo from the video. It results in loss of quality + meta/exif data which was present in the original image.

**Additional context**
For documentation, we can refer to the following implementation
- [Github: LimitPoint/LivePhoto](https://github.com/LimitPoint/LivePhoto/blob/master/LivePhoto.swift#L30)
- [Apple: Capturing Still and live photos](https://developer.apple.com/documentation/avfoundation/photo_capture/capturing_still_and_live_photos/capturing_and_saving_live_photos?language=objc)

**Comment by AlexV525** [comment]
This is scheduled in our milestone as a goal. Perhaps it can be released along v3.0.0.

**Comment by ua741** [comment]
Hey @AlexV525 , 
Whenever you get a chance, can you please review the linked PR? 
Thanks!
